bcdaworker/queueing/enqueue.go

```diff
@@ -58,3 +60,19 @@ func (q queEnqueuer) AddJob(job models.JobEnqueueArgs, priority int) error {
 
 	return q.Enqueue(j)
 }
+
+// ALR ENQ...
+func (q queEnqueuer) AddAlrJob(job models.JobAlrEnqueueArgs, priority int) error {
```

Comment:
Should we add this to enqueue_test.go?

Reply:
Yup! But it's going to be in your PR (https://github.com/CMSgov/bcda-app/pull/660/files#diff-6142142004eef8af8589b36ab24ec666a3f15b4e15593a2fa928038fbed7af62R169)